sample/.github/workflows/test.yml

```
@@ -0,0 +1,50 @@
+name: Test
+on:
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+
+      # TODO: More CI steps will be added in https://github.com/nimblehq/kmm-templates/issues/4
+
+  generate_sample_project:
+    # The job "generate_sample_project" identify that the job "test" must complete successfully
+    # before this "generate_sample_project" job will run
+    needs: test
+    name: Generate the sample project
+    runs-on: macOS-12
+    timeout-minutes: 30
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          submodules: 'recursive'
+
+      - name: Remove the old sample project
+        run: rm -rf sample
+
+      - name: Generate the new sample project
+        run: ./make.sh --bundle-id co.nimble.kmm.sample --bundle-id-staging co.nimble.kmm.sample.staging --project-name sample --ios-version 14.0
```

```suggestion
        run: ./make.sh --bundle-id co.nimble.kmm.sample --bundle-id-staging co.nimble.kmm.sample.staging --project-name sample --ios-version ${IOS_VERSION}
```
Can we put this to be the variable?

---

@lydiasama I raised my concern on this PR https://github.com/nimblehq/kmm-templates/pull/24/files#r1295466511. Let @phongvhd93 handle this later 🙏